### ISSUE TITLE
Ensure content that is intended to be hidden from all users is not rendered by assistive technology

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#75]
## Description
Often CSS, JavaScript, and other DHTML techniques are used to hide or collapse content that is on-screen. This content can also be content that is meant to be dynamically available or hidden based on selected options. This content can sometimes be a small piece of text, a set of links, or another webpage. Assistive technology such as screen readers may render this content to the user if the content is not marked in a certain manner as "hidden". Having the wrong content rendered may cause users to read incorrect information or even activate links that are off screen. This can lead to confusion and incorrect results.

A new attribute titled hidden is provided in HTML5. This attribute indicates that content should be hidden from all users. This is a great structural way to indicate hidden content without relying on CSS. This attribute however is not supported by all assistive technologies and thus pages should continue to use CSS properties such as display:none in addition to the hidden attribute to hide content.

### Header
- [x] Main menu nav items should not be visible to screen reader unless Screen Reader User opens the menu.  Can use `display: none`.  Although, if possible, it would also be nice to preserve the slide on/off css animation of the mobile main menu.

![623483184af08659fedcf838b4432cae218c55ed-1641278499](https://user-images.githubusercontent.com/8166060/153090802-833539dc-a872-40b9-9462-769868f7b135.png)

### Sort/filter/search bar

- [x] "Filter" and "Sort by" menu items should not be visible to screen reader unless Screen Reader User opens the menu.  can use `display: none`.  Although, if possible, it would also be nice to preserve the expand/collapse css animation of the those dropdown menus.

![44eaea17060c5f439f4aff7a990ad69024a24829-1641289883](https://user-images.githubusercontent.com/8166060/153091086-a0578b65-5a20-4498-8b70-eea86bf1fa57.png)

## CIC Estimate
0 points

**Comment by dananjohnson** [comment]
Header nav visibility resolved by https://github.com/lsst-epo/rubin-obs-client/commit/105a252dd754bf23fb0529cde0dad906ca96173c.

**Comment by dananjohnson** [comment]
@blnkt we still need to fix the filter & sort menus.
